test/providers/provider_level_zero.cpp

```diff
@@ -167,21 +162,14 @@ struct umfLevelZeroProviderTest
         }
 
         ASSERT_NE(zeMemoryTypeExpected, ZE_MEMORY_TYPE_UNKNOWN);
-
-        memAccessor = accessor;
     }
 
-    void TearDown() override {
-        int ret = destroy_context(hContext);
```

Comment:
where do you call destroy context?

Reply:
`l0Params_device_memory`, `l0Params_shared_memory`, `l0Params_host_memory` are global variables, see L306 - L311. So it is an error to destroy the context in the `TearDown` method because it is called after each test.

The same is true for the CUDA tests